### PR TITLE
added kibana log url

### PR DESCRIPTION
**Pull request by dguendisch**
**What this PR does / why we need it**:
Adds a direct link to the metadata of each testrun and teststep pointing to the respective kibana logging installation with a an active filter on the specific testrun.

### CODE REVIEW

File: pkg/testrunner/result/output.go [hunk 1]
```diff
@@ -268,3 +276,23 @@ func getOSConfig(tmClient kubernetes.Interface, namespace, minioEndpoint string,
 		BucketName: minioConfig.Data["objectstore.bucketName"],
 	}, nil
 }
+
+// buildKibanaWorkflowURL construct a valid kibana url from the given endpoint and workflow
+// typical endpoint: https://kibana.ingress.<my-cluster-domain>
+// typical kibana url: https://kibana.ingress.<my-cluster-domain>/app/kibana#/...
+func buildKibanaWorkflowURL(endpointPath string, namespace string, workflow string) string {
+	const IndexPatternID = "d1134060-5189-11e9-b7dc-3fcc67c9fd25"
```
Comment (schrodit) [inline pkg/testrunner/result/output.go]: nit: lowercase variable name `indexPatternID`

File: pkg/util/util.go [hunk 2]
```diff
@@ -169,3 +170,16 @@ func PrettyPrintStruct(obj interface{}) string {
 	}
 	return string(str)
 }
+
+// MarshalNoHTMLEscape is nearly same as json.Marshal but does NOT HTLM-escape <, > or &
+// However it does add a newline char at the end (as done by json.Encoder.Encode)
+func MarshalNoHTMLEscape(v interface{}) ([]byte, error) {
```
Comment (schrodit) [inline pkg/util/util.go]: Maybe include JSON in the function name to specify that this method Mashals json data.

File: cmd/testrunner/cmd/runtestrun/run_testrun.go [hunk 3]
```diff
@@ -90,6 +90,7 @@ var runTestrunCmd = &cobra.Command{
 		run := []*testrunner.Run{
 			{
 				Testrun: &tr,
+				Metadata: &testrunner.Metadata{},
```
Comment (schrodit) [inline cmd/testrunner/cmd/runtestrun/run_testrun.go]: Thanks for the fix;)

File: pkg/testrunner/result/output.go (hunk 1 above)
Comment (dguendisch) [reply pkg/testrunner/result/output.go]: you're right, changed it (confused it with the uppercase requirement for the respective struct for json marshalling)

File: pkg/util/util.go (hunk 2 above)
Comment (dguendisch) [reply pkg/util/util.go]: as discussed per slack, leaving it like that for now

**Comment by dguendisch** [comment]
applied changes, ptal

**Pull request merged**